tests/NLog.UnitTests/Layouts/SimpleLayoutParserTests.cs

```diff
@@ -813,14 +813,12 @@ public void Parse_AppDomainFixedOutputWithRawValue_ConvertSingleToLiteralAndKeep
             var layout = (SimpleLayout)Layout.FromString(input);
 
             // Assert
+            Assert.True(layout.IsFixedText);
             var single = Assert.Single(layout.Renderers);
-            var singleRaw = Assert.IsType<LiteralWithRawValueLayoutRenderer>(single);
-            var succeeded = singleRaw.TryGetRawValue(LogEventInfo.CreateNullEvent(), out var rawValue);
+            Assert.IsType<LiteralWithRawValueLayoutRenderer>(single);
+            var succeeded = layout.TryGetRawValue(LogEventInfo.CreateNullEvent(), out var rawValue);
             Assert.True(succeeded);
-            var rawValueInt = Assert.IsType<int>(rawValue);
```

Comment:
@snakefoot  Why is the assert of int type not needed anymore?

Reply:
The cast will fail. But the main test is the output from `layout.TryGetRawValue()` instead of checking the result of the internal renderer.